context/src/main/java/io/opentelemetry/context/internal/InstrumentationUtil.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.context.internal;
```

Comment:
I think this should live in `opentelemetry-api` instead of `opentelemetry-context`. `opentelemetry-context` can function as a standalone context propagation API without any knowledge of opentelemetry or instrumentation.

Reply:
done in da8ad454e14ae6c53f333459dc1779e668f2989f